<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Amir</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="table_frame" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>521</width>
      <height>697</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Terminal</string>
     </attribute>
     <widget class="QTextEdit" name="compile_box">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>521</width>
        <height>501</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Compilation Output</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(1, 36, 86);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>503</y>
        <width>513</width>
        <height>158</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QComboBox" name="prog_type_combo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>C++</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Python</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Matlab</string>
        </property>
       </item>
      </widget>
      <widget class="QFrame" name="questions_frame">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>191</width>
         <height>111</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="run_push_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>120</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
      <widget class="QPushButton" name="open_code_push_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>10</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Open Code</string>
       </property>
      </widget>
      <widget class="QPushButton" name="compile_push_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>90</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Compile</string>
       </property>
      </widget>
      <widget class="QPushButton" name="open_report_push_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Open Report</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Folder</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTreeView" name="folder_tree_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>491</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
